src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/staves/FireStaff.java

```diff
@@ -47,13 +48,14 @@ public ItemUseHandler getItemHandler() {
             Player p = e.getPlayer();
             ItemStack item = e.getItem();
             Location loc = p.getTargetBlock(null, 7).getLocation();
+            Block block = loc.getBlock();
```

Comment:
why do you imitate this here its not used anywhere here

Reply:
he used it as pass by value in the code